### DISCUSSION TITLE
Hi need some legal query help about hacking and running python programs

### DISCUSSION THREAD

**Discussion by azanej24** [community/community#67501] [New to GitHub]
Hi,
I wanted to download python software and use it to emulate a phone number that isn't recognised meaning that the phone number wouldn't be owned by someone, I have also done a phone UK search and verified that nobody has the phone numbers on their contract or possession, I need access to these old phone numbers and will be simulating their usage using a microsoft laptop that I will buy from currys PC world or other store, 

I have been advised to talk to microsoft IT legal department to verify that this isn't against the computing misuse act and doesn't validate as me hacking without consent to obtain information of other people in order to gain access or store or use data for my own right of use. I will simply be using the programmes to simulate a phone number in order to receive texts, I will use these texts to verify my 2nd verification into my previous work/business/school accounts. As previously mentioned these numbers aren't being used by other people right now.

I didn't want to commit a felony or anything agaisnt the computer misuse act as I cannot protect myself properly agaisnt the police or local enforcement in that regard, and wanted to make sure that if I use a computer that has its software security updated that I can be assisted by microsoft IT department if the computer is hacked or installs a virus that the antivirus software doesn't defend agaisnt; (so preventing me from buying new laptops that fail to install the programmes). Also these are my previous phone numbers.

**Comment by queenofcorgis** [comment]
Hi @azanej24 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Microsoft like their community: https://answers.microsoft.com/en-us. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by GhostOf0days** [comment]
Sounds like you're trying to create a VoIP number. Many services don't allow you to use VoIP numbers to verify. Also, if you're trying to spoof someone's number, it may be illegal depending on where you live.
